Spring level Update

leiqian-nmsu opened the pull request:
Hi all, here is the spring level update. Please check it. Thanks!

leiqian-nmsu committed:
Prepare for spring level

leiqian-nmsu committed:
tile-additions new

leiqian-nmsu committed:
Merge branch 'master' into spring-level

# Conflicts:
#	Alchemic Forest Game/Assets/Tile Palettes/Terrain Tiles.prefab
#	Alchemic Forest Game/Library/assetDatabase3
#	Alchemic Forest Game/Library/metadata/00/00000000000000003000000000000000

leiqian-nmsu committed:
04/27/2018 Modified spring-level

leiqian-nmsu committed:
04/27/2018 additional

leiqian-nmsu committed:
Merge branch 'master' into spring-level

# Conflicts:
#	Alchemic Forest Game/.vs/Alchemic Forest Game/v15/Server/sqlite3/storage.ide
#	Alchemic Forest Game/.vs/Alchemic Forest Game/v15/sqlite3/storage.ide
#	Alchemic Forest Game/Assets/Animations/Bonfire.meta
#	Alchemic Forest Game/Library/CurrentLayout.dwlt
#	Alchemic Forest Game/Library/InspectorExpandedItems.asset
#	Alchemic Forest Game/Library/assetDatabase3
#	Alchemic Forest Game/Library/expandedItems
#	Alchemic Forest Game/Library/metadata/00/00000000000000003000000000000000
#	Alchemic Forest Game/ProjectSettings/TagManager.asset

leiqian-nmsu committed:
04/28/2018 spring-level fishInRiver

leiqian-nmsu committed:
05012018 modified

leiqian-nmsu committed:
Merge branch 'master' into spring-level

# Conflicts:
#	Alchemic Forest Game/.vs/Alchemic Forest Game/v15/sqlite3/storage.ide
#	Alchemic Forest Game/Assets/Prefabs.meta
#	Alchemic Forest Game/Library/CurrentLayout.dwlt
#	Alchemic Forest Game/Library/assetDatabase3
#	Alchemic Forest Game/Library/expandedItems
#	Alchemic Forest Game/Library/metadata/00/00000000000000002000000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000003000000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000004000000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000004100000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000005000000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000005100000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000006000000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000006100000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000007000000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000007100000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000008000000000000000
#	Alchemic Forest Game/Library/metadata/00/00000000000000009000000000000000
#	Alchemic Forest Game/Library/metadata/00/0000000000000000a000000000000000
#	Alchemic Forest Game/Library/metadata/00/0000000000000000a100000000000000
#	Alchemic Forest Game/Library/metadata/00/0000000000000000b000000000000000
#	Alchemic Forest Game/Library/metadata/00/0000000000000000c000000000000000

leiqian-nmsu committed:
05032018 Spring Level

leiqian-nmsu committed:
05032018(2)

leiqian-nmsu committed:
Merge branch 'crafting' of https://github.com/SDev-2018/Alchemic-Forest into crafting

# Conflicts:
#	Alchemic Forest Game/.vs/Alchemic Forest Game/v15/sqlite3/storage.ide
#	Alchemic Forest Game/Library/CurrentLayout.dwlt
#	Alchemic Forest Game/Library/assetDatabase3
#	Alchemic Forest Game/Library/metadata/00/00000000000000003000000000000000

leiqian-nmsu committed:
05/04/2018 Spring Level Update:
(1) 5 CircleRunes: Change color from blue to red. (3 circles are for bridges, 1 circle is for fishingnet, 1 circle is for exit)
(2) 3 bridges and CoverBridge tilemap
(3) Axes
(4) Test fishing mode: trigger fishingSymbol one time to get one fish, trigger twice to get two fishes.
(5) Deactivate 2 cats after they get fishes.
BTW, all woods and cloth and diamonds, they are hided behind trees or under tree crowns.

leiqian-nmsu commented:
05/04/2018 Spring Level Update:
(1) 5 CircleRunes: Change color from blue to red. (3 circles are for bridges, 1 circle is for fishingnet, 1 circle is for exit)
(2) 3 bridges and CoverBridge tilemap
(3) Axes
(4) Test fishing mode: trigger fishingSymbol one time to get one fish, trigger twice to get two fishes.
(5) Deactivate 2 cats after they get fishes.
BTW, all woods and cloth and diamonds, they are hided behind trees or under tree crowns.

Mdaviet reviewed (changes requested):
Please add all the things you mentioned in the comments, and then maybe a bit more.  The level looks great, but it doesn't play properly.  I think you will want to take the player prefab from the test environment so the player works better, and there currently is no way to progress at all.  You start in the lower left corner, and can't do anything because there is nothing.

leiqian-nmsu committed:
05062018 Spring Level update

leiqian-nmsu committed:
05062018 spring level
1. netRune pick up problem
2. two bridges
3, inventory and dialogue
4, exit Rune
5. torch

leiqian-nmsu committed:
Fixed issues with player not interacting as required.

leiqian-nmsu committed:
Add level boundries to prevent looking out at nothing.

leiqian-nmsu committed:
05072018 spring level

leiqian-nmsu committed:
05072018 spring level addtion

leiqian-nmsu committed:
05072018 Update (Bridge and Axes waited to be fixed)
